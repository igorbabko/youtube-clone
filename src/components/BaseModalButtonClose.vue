<template>
  <button class="p-2 focus:outline-none" data-testid="base-modal-button-close">
    <BaseIcon name="x" />
  </button>
</template>

<script>
import BaseIcon from './BaseIcon.vue'

export default {
  components: {
    BaseIcon
  }
}
</script>
