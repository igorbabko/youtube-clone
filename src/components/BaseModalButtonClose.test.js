import { render } from '@testing-library/vue'
import BaseModalButtonClose from './BaseModalButtonClose.vue'

it('renders correctly', () => {
  const { html } = render(BaseModalButtonClose)

  expect(html()).toMatchSnapshot()
})
