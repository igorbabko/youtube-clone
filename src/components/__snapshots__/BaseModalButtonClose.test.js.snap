// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
"<button class=\\"p-2 focus:outline-none\\" data-testid=\\"base-modal-button-close\\"><svg class=\\"w-6 h-6\\" fill=\\"currentColor\\" viewBox=\\"0 0 20 20\\" xmlns=\\"http://www.w3.org/2000/svg\\" data-testid=\\"base-icon\\">
    <path fill-rule=\\"evenodd\\" d=\\"M4.293 4.293a1 1 0 011.414 0L10 8.586l4.293-4.293a1 1 0 111.414 1.414L11.414 10l4.293 4.293a1 1 0 01-1.414 1.414L10 11.414l-4.293 4.293a1 1 0 01-1.414-1.414L8.586 10 4.293 5.707a1 1 0 010-1.414z\\" clip-rule=\\"evenodd\\"></path>
  </svg></button>"
`;
